zuul.d/project-templates.yaml

```diff
@@ -60,6 +60,7 @@
         - podified-multinode-edpm-deployment-crc: *content_provider
         - cifmw-crc-podified-edpm-baremetal: *content_provider
         - podified-multinode-hci-deployment-crc: *content_provider
+        - podified-multinode-edpm-deployment-crc-tempest: *content_provider
```

Comment:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/tempest_multinode.yaml#L118 has the files clause. Do we want this running on all files?

Reply:
let me add this in separate PR.